lib/drivers/animasDriver.js

```diff
@@ -955,8 +958,11 @@ module.exports = function (config) {
   };
 
   var decodeDate = function(encoded) {
-    var year = (encoded.monthYear & 0x0f) + 2007; // OneTouch Ping starts in year 2007 as 0
-    // TODO: Animas Vibe starts in 2008
+    var startYear = 2007; // OneTouch Ping starts in year 2007 as 0
+    if(modelNumber === 'IR1295') {
```

Comment:
Since we know we end up with a lot of unusual situations in the wild, would it be useful to be exact in what we expect and fail otherwise?  Be defensive, that is. For example:

```
var startYear;
if (modelNumber === 'IR1285') {   // Ping
  startYear = 2007;
} else if (modelNumber === 'IR1295') {  // Vibe
  startYear = 2008;
} else {
  // Unexpected, so fail fast here...
}
```